pallets/ddc-customers/src/migration.rs

```diff
@@ -39,8 +39,35 @@ pub mod v0 {
 	>;
 }
 
+pub mod v1 {
+	use frame_support::pallet_prelude::*;
+
+	use super::*;
+
+	#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
+	pub struct Bucket<AccountId> {
+		pub bucket_id: BucketId,
+		pub owner_id: AccountId,
+		pub cluster_id: ClusterId,
+		pub is_public: bool,
+		pub is_removed: bool,
+	}
+
+	#[storage_alias]
+	pub(super) type BucketsCount<T: Config> = StorageValue<crate::Pallet<T>, BucketId, ValueQuery>;
+
+	#[storage_alias]
+	pub(super) type Buckets<T: Config> = StorageMap<
+		crate::Pallet<T>,
+		Twox64Concat,
+		BucketId,
+		Bucket<<T as frame_system::Config>::AccountId>,
+		OptionQuery,
+	>;
+}
+
 // Migrate to removable buckets
-pub fn migrate_to_v1<T: Config>() -> Weight {
+pub fn migrate_from_v0_to_v2<T: Config>() -> Weight {
```

Comment:
You shouldn't change the name of previous migration that was provided for the `v0` -> `v1` update. You should leave it as is, and if there is a new field in the `Bucket` (`total_customers_usage`) and the old migration is not compiling because of the missing field, define a new struct `BucketV1` in the scope of `v1` migration and use it across the `v1` migration. 
The idea is that every migration shows the transition of an entity (`Bucket`) that was planned for this migration. In the end, you will have `BucketV0`, `BucketV1`, `BucketV2`, `BucketVn` that will allow you to understand how the data structure has been changing through the time.

The `migrate_to_v1`, i.e. `v0` -> `v1` migration has been executed already in the Mainnet, it will not run the second time due to the `on_chain_version` constraint. 

Only the new `migrate_from_v1_to_v2` will be executed after specifying it in the `Migrations` tuple.